modular_zubbers/code/modules/mod/modules.dm

```diff
@@ -55,3 +55,59 @@
 	bullet = 10
 	laser = 10
 	energy = 15
+
+#define WEARER_CONTROL 0
+#define AI_CONTROL 1
+
+/obj/item/mod/module/mind_swap
+	name = "MOD Neural Transference module"
+	desc = "Swaps the MOD wearer's and Assistant AI's neural pathways."
+	removable = FALSE
+	required_slots = list(ITEM_SLOT_FEET, ITEM_SLOT_GLOVES, ITEM_SLOT_OCLOTHING, ITEM_SLOT_HEAD)
+	module_type = MODULE_ACTIVE
+	//Who's in control of the wearer's body
+	var/controller = WEARER_CONTROL
```

Comment:
you could make this into a boolean, something like ai_control = FALSE, makes the switch logic lower down easier as well.